lib/HTTP/Status.pm

```diff
@@ -55,6 +55,8 @@ my %StatusCode = (
     416 => 'Request Range Not Satisfiable',
     417 => 'Expectation Failed',
     418 => 'I\'m a teapot',		      # RFC 2324
+    420 => 'Policy Not Fulfilled',
+    421 => 'Too Many Connections From Your IP Address',
```

Comment:
RFC 7540 seems to indicate that `421` is "Misdirected request". https://tools.ietf.org/html/rfc7540#page-66